molecule/default/tests/test_default.py

```diff
@@ -1,14 +1,49 @@
 import os
-
+import pytest
 import testinfra.utils.ansible_runner
 
+
+"""ASC-240: Verify the requested glance images were uploaded
+
+RPC 10+ manual test 10
+"""
+
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('compute-infra_hosts')[:1]
+
+# attach the utility container on infra1
+pre_cmd = "lxc-attach -n `lxc-ls -1 | grep utility | head -n 1` -- bash -c "
```

Comment:
Can we rename this variable to reflect the utility attachment so that the context of the command is clear?

Reply:
Yes, I totally agree